cypress/support/e2e.ts

```diff
@@ -24,6 +24,7 @@ beforeEach(() => {
 
     cy.readFile('dist/recorder.js').then((body) => {
         cy.intercept('**/static/recorder.js*', { body }).as('recorder')
+        cy.intercept('**/static/recorder-v2.js*', { body }).as('recorder')
```

Comment:
Fun! We were only testing the old rrweb recorder previously 🙈 Added this which fixed the cypress load error but now the tests seem flakey. @daibhin might need rewriting / investigating what is different...

Reply:
That said - tests pass so 🤷

Reply:
I ran the Cypress tests locally a number of times and all seemed fine. Happy to ship if you are

Reply:
👍